docs/dashboards/chart-builder.md

```diff
@@ -0,0 +1,176 @@
+alias can describe for example availability zone, cpu type, etc. Read more
+about [tag aliases](/monitoring/tag-aliases-support).
+
+You can filter by tag aliases the same way as they are regular tags so this is completely
+transparent to you.
+
+You can use [variables](#variables) as tag values so they can be configured dynamically
+from the report level.
+
+## Group by
+
+In Group by section you choose what grouping data series on a graph. For example, if you
+choose `os.host` tag and aggregation in `Aggregation` field you choose `all separately`
+the chart will show data series for each host separately. Each data series is made up of
+the selected metric on a particular host.
+
+You can use [variables](#variables) as tag values so they can be configured dynamically
+from the report level.
+
+### Aggregation functions
+
+Back to the example from the section above, if you choose aggregation function instead of `all separately`
+only one data series is created for all the hosts. The result depends on the aggregation function
+function you choose. The aggregation function determines how the metrics are
+aggregated into a single data series. You can pick one of `avg`, `sum`, `min`, `max`
+
+
+## Rollup by
+
+Datapoints sent to Sematext have time label. If a metric is updated every 10 second, and you are
+looking at 24 hours time range, you need 14400 points to display all datapoints. 
+Presenting so many datapoints on the chart would make the chart not readable and would impact
+the web browser performance. If we build time buckets each containing 600 datapoints
+we will have 24 datapoints to present on the chart (each datapoint for each hour). Rollup function
+decides how time buckets are built (what aggregation is performed).
+
+Each of predefined metrics has its own default rollup function defined depending on the type of
+a metric. All the gauge metrics use average rollup function and all metrics of a type counter
+use sum rollup function. In some cases it makes sense to customize the setting.
+
+_NOTE: Recommended for expert users._
+
+
+## Variables
+
+Variables are used to dynamically filter and group by one or more widgets in the report by tags.
+
+Filters from the `Filter by` section don't have to be configured statically when a chart
+is created. Sometimes you need charts on the dashboard to react on a global filter
+change. Consider a case where you're troubleshooting a system and you realized that one of the
+hosts has problems. You have a dashboard which contains aggregate information for all hosts. You can
+of course reconfigure all the components from the dashboard so they show only interesting host data.
+As you might expect this is not a very efficient way of troubleshooting. Here is where Variables
+become a very useful feature. You can just create a tag variable which points to field `os.host`.
+Later this variable should be added to the charts which should listen for variable changes.
```

Comment:
Later this variable should be added to the charts which should listen for variable changes.
=>
This variable should later be added to the charts that listen for variable changes.